packages/teams-js/src/public/dialog.ts

```diff
@@ -111,12 +132,30 @@ export namespace dialog {
       ensureInitialized(FrameContexts.content, FrameContexts.sidePanel, FrameContexts.task, FrameContexts.meetingStage);
       sendMessageToParent('tasks.updateTask', [dimensions]);
     }
+
+    /**
+     * Checks if dialog.update capability is supported currently
+     *
+     * @returns boolean to represent whether dialog.update is supported
+     */
     export function isSupported(): boolean {
       return runtime.supports.dialog ? (runtime.supports.dialog.update ? true : false) : false;
     }
   }
 
+  /**
+   * Namespace to open the dialog using bot
+   */
   export namespace bot {
+    /**
+     * Allows an app to open the dialog module using bot.
+     *
+     * @param botUrlDialogInfo - An object containing the parameters of the dialog module including completionBotId.
```

Comment:
Does this link to the BotUrlDialogInfo docs and are those docs finished/sufficient?

Reply:
(In general, same feedback as for dialog.open above)